src/content/pages/privacy-guides/vpn-latency-explained.md

```diff
@@ -55,7 +55,7 @@ In practice, the latency is slightly increased by each equipment on the network
 
 ## Bandwidth
 
-Another important speed metric for a network is its **bandwidth**, which is the maximum speed a network can sustain for a given period of time. Bandwidth is usually expressed in megabits per second (Mbps), but as optical fiber and 5G are getting more widespread, it is getting common to express it in gigabits per second (Gbps).
+Another important speed metric for a network is its **bandwidth**, which is the maximum speed a network can sustain for a given period of time. Bandwidth is usually expressed in megabits per second (Mbit/s), but as optical fibre and 5G are getting more widespread, it is getting common to express it in gigabits per second (Gbit/s).
```

Comment:
Mbps and Gbps are more common when ISP or service providers are communicating with clients, it should stay like this